specs/WorkerPostMessage.md

```diff
@@ -0,0 +1,682 @@
+Post message support for Dedicated and Service Workers
+===
+
+# Background
+Currently, if developers want to post a message to or from a worker, they must
+first post a message to the main thread and then post a message to the worker
+from the main thread. This leads to increased load on the main thread.
+
+The WebView2 team is introducing APIs that allow messages to be posted directly
+from the Host app to dedicated and service workers, and vice versa. This
+eliminates the need for the main thread to act as an intermediary, thereby
+improving app performance and responsiveness.
+
+# Description
+We propose the following APIs:
+
+**PostWebMessageAsJson**: This API gives ability to post a message to

```

Comment:
These signatures intentionally align with PostWebMessageAs* from CoreWebView2 that offer the equivalent functionality on the main thread, correct?

Should documentation cross-link?
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2?view=webview2-dotnet-1.0.3124.44